### ISSUE TITLE
Syntax linting of import causes NullPointerException

### ISSUE THREAD

**Issue by zoren** [clj-kondo/clj-kondo#2013]
v2023.02.17

native

CLI

For the invalid Clojure snippet `(import "")` I get an NPE. I would have expected a more descriptive error message.

```
❯ echo '(import "")' | clj-kondo --lint -
<stdin>:0:0: error: Can't parse <stdin>, java.lang.NullPointerException
linting took 39ms, errors: 1, warnings: 0
```

```
(import "")
```
output:
```
<stdin>:0:0: error: Can't parse <stdin>, java.lang.NullPointerException
```

I would have expected an error message saying this is not valid Clojure.

I would want to try and fix this myself.

**Comment by borkdude** [comment]
PR welcome!
